cowprotocol/explorer
# [Batch Viewer] Differentiate SELL and BUY orders

# Summary

Closes #45 

Updated batch viewer logic in order to add order kind to differentiate between SELL and BUY orders.

<img width="925" alt="Screen Shot 2022-06-06 at 22 37 47" src="https://user-images.githubusercontent.com/11525018/172304640-d048e4e1-0a4a-4bcd-8f84-f33705038b76.png">

# To Test

1.  Open tx page and go to the batch diagram view. (i.e : `tx/0xc2eb04b549d68c770062591e2561c5b1534ea665c6179d33a779c52376c9161d`)
* You will see red edges for SELL orders and GREEN edges for BUY ones.
